website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
IMO adding company copyright + author is a bit excessive, and is going to begin to make our awesome short headers like the old long ones again. I know this was in the 8.0 & I'm not blocking for it on this PR, but I'm wondering if we're setting a bad precedence here?

@pedrobaeza - am I being overly anal here? It seems to me this was the intent of removing the author header, and instead focusing towards the contributors of the ReadMe.